stores/src/main/java/com/issoft/training/stores/utils/RandomStorePopulator.java

```diff
@@ -0,0 +1,60 @@
+package com.issoft.training.stores.utils;
+
+import com.github.javafaker.Faker;
+import com.issoft.training.domain.shop.Product;
+import com.issoft.training.domain.shop.categories.Categories;
+import org.reflections.Reflections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RandomStorePopulator {
+    private static final Faker faker = new Faker();
+
+    private static Product createProductByCategoriesName(String categoryName) {
+        int price = (int) faker.number().randomNumber();
+        int rate = (int) faker.number().randomNumber();
+        switch (categoryName) {
+            case "food":
+                return new Product(faker.food().vegetable(), price, rate);
+            case "animal":
+                return new Product(faker.animal().name(), price, rate);
+            case "book":
+                return new Product(faker.book().genre(), price, rate);
+            case "beer":
+                return new Product(faker.beer().name(), price, rate);
+            case "color":
+                return new Product(faker.color().name(), price, rate);
+            default:
+                return new Product(faker.name().firstName(), price, rate);
```

Comment:
why not to use switch just for creating a name, and later (after switch) create a new Product, new Product(name, faker.number.random...)